### ISSUE TITLE
Feedback - Improve the lesson preview CTA banner

### ISSUE THREAD

**Issue by Clorith** [WordPress/Learn#2864]
## Type of feedback
// dev

## Description
Users may think they are required to have an account on WordPress.org to take a lot of the lessons right now, due ot the wording of the banner when previewing lessons.

Being signed in should only be a requirement if you wish to track your progress, or take the quizzes associated with a course.

By updating the text from **"Register or sign in to take this lesson."** to something a bit more descriptive such as **"Register or sign in to track your progress, and participate in lesson quizzes."** we can avoid confusion, and reduce unnecessary signups on WordPress.org.

I included a secondary alternative, although I think it adds a bit too much text and can be a bit more confusing, I think it's an interesting idea to just get a conversation going on a potential second iteration perhaps. This adds in a byline to make it explicitly clear that an account is not needed with the wording **"This is a public lesson, you do not need an account to access this course."**

## Screenshots or screen recording (optional)
![Current banner](https://github.com/user-attachments/assets/fcb5045c-e3c2-4eb5-a5e8-0b68f1844278)
The current banner indicates you need to register or sign in to take this lesson.

![proposed new look with new wording](https://github.com/user-attachments/assets/3b593f2a-f9d5-4c7c-bc1d-ed6086f8e9ea)
The banner with the proposed new text.

![second proposal concept, explicitly declaring an account is not needed](https://github.com/user-attachments/assets/a40bf1e9-c5dc-43c9-a2f9-ad08dae10e06)
This is merely a concept proposal on making it more explicit that an account isn't needed, but this may as noted add some confusion?

**Comment by kathrynwp** [comment]
This came up [in Slack](https://wordpress.slack.com/archives/C02RW657Q/p1724880201429809), linking here for full context.

**Comment by adamwoodnz** [comment]
@fcoveram @thetinyl any thoughts on this proposal?

**Comment by thetinyl** [comment]
Oh, interesting. Thanks for calling attention to this @Clorith. I do see what you mean. 

Some questions that came up for me:
- Looking at the banner as a whole even with the suggested tweaks, we still have the 'This is a preview lesson.' language. To me, this implies that there's more to the course I don't have access to... even if we state clearly what registration offers. It's not really a "preview". @adamwoodnz How closely is the content tied to Sensei vs. what elements of the banner can we change?
- The impression I got from working on some of the Learn redesign efforts was that the Training team wanted to encourage course registration. @kathrynwp may be able to weigh in here, but if this _is_ the case, it does add another layer of consideration for the language we use.

**Comment by kathrynwp** [comment]
@thetinyl See #2829 for an issue related to the "preview" wording!

> The impression I got from working on some of the Learn redesign efforts was that the Training team wanted to encourage course registration. @kathrynwp may be able to weigh in here, but if this is the case, it does add another layer of consideration for the language we use.

I do think it's preferable if people register for the course rather than just preview its lessons. It allows us to gather feedback when they complete the course, and gives them a record of having taken the course on their WordPress.org profile. It also allows them to get all the benefits of going through the course sequentially, including the interactive exercises like quizzes. The previews are a nice-to-have option that we offer, but overall it's a better experience for everyone if they register.

**Comment by thetinyl** [comment]
So I've been noodling over this one. 

As @Clorith points out, I think there's value in being descriptive to provide clarity on what added benefit logging in/registering provides. I also think there's a balance we need to strike with the Training team's goals and staying true to the complete Learn experience. 

Is it fair to claim that, while someone can see almost all the lesson content in "preview mode" (pulling language from @fcoveram's proposal in #2869)... they can't actually **complete** the lesson without logging in (they can't track and they'll be missing quizzes and other interactive bits)? 

If this is the case, my suggestion for updating the language would be something like:

> **Register or log in to track progress and complete this lesson.**

**Comment by fcoveram** [comment]
Question to @thetinyl 

> If this is the case, my suggestion for updating the language would be something like: Register or log in to track progress and complete this lesson.

Is it ok to swap "register" with the "log in" words to match the design? Here is a screenshot of how it would look like on the page.

![screenshot of the lesson before users log in](https://github.com/user-attachments/assets/0f7b4341-9860-49c3-ac18-61c6679c9ad9)

On the other hand, after users log in to the WordPress site, they need to "take the lesson" to see the content and start tracking the progress. What could be a good copy for both the message heading and the CTA? Here is a screenshot from the [design page](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=5492-15422&node-type=SECTION&m=dev) in Figma.

![screenshot of the lesson after users logged in](https://github.com/user-attachments/assets/cc076ee5-2ec0-44bc-8d56-05d0d88fa7d3)

If you compare this screen with the previous one, this one has the admin bar showing the user info.

**Comment by thetinyl** [comment]
> Is it ok to swap "register" with the "log in" words to match the design?

@fcoveram Absolutely! 

As for the other message... that's a tricky one. Especially if a logged in user is just previewing the lesson content (which I assume works the same way for both user states?)

Maybe something like:

> Take this course to track your progress and complete the materials. 
> BUTTON: Take course

**Comment by fcoveram** [comment]
> …which I assume works the same way for both user states?

Yes. Logged and not logged in users will land on a page with the message and no visible content related to the lesson.

The copy suggested works well. I will update the content 🙂
